### PR TITLE
[WIP] #17 put all tokens to env variables

### PR DESCRIPTION
**Pull request by bimlendra**
@hyzhak : You can suggest me what other values required to be store in firebase config.

### CODE REVIEW

File: .runtimeconfig.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "dashbot": {
+    "key": "54mlQ1bEx6WFGlU4A27yHZubsQXvMwYPAqHtxJYg"
```
[the remark this answers is not in the document]
Comment (hyzhak) [reply .runtimeconfig.json]: ok, but why do we use `54mlQ1bEx6WFGlU4A27yHZubsQXvMwYPAqHtxJYg` and `54bcfb2a-a12b-4c6a-8729-a4ad71c06975` in our source code ('./index.js`) as well?

**Comment by bimlendra** [comment]
@hyzhak : Can I merge this with master now?

File: functions/tests/.runtimeconfig.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (hyzhak) [inline functions/tests/.runtimeconfig.json]: those private keys are still here

**Comment by bimlendra** [comment]
@hyzhak These are fake keys at functions/tests/.runtimeconfig.json

**Comment by hyzhak** [comment]
@bimlendra if they are fake why do you have attached them to .gitignore?

**Comment by hyzhak** [comment]
@bimlendra If `runtimeconfig` is fake you don't need to put it in gitignore

**Comment by bimlendra** [comment]
@hyzhak : That check was for .runtimeconfig in root folder. I forgot to be specific. Updated again. However, tests are getting failed here on github but these all are pass at my console.

**Comment by hyzhak** [comment]
@bimlendra you don't need to ignore `runtimeconfig` if it is just a fake keys for test purpose. And btw it would be needed for the rest of team as well.

**Comment by bimlendra** [comment]
I have ignored 
**functions/.runtimeconfig.json**, cause anyone might execute command to get 
**functions/.runtimeconfig.json** from **firebase** & it might pushed to git. So, to avoid such incident, I have added 
**functions/.runtimeconfig.json** in **gitignore**.

However, test config i.e.
**functions/tests/.runtimeconfig.json** is still available on git. Does it make sense?

**Comment by bimlendra** [comment]
@hyzhak : Can we merge this with master now?

**Comment by hyzhak** [comment]
@bimlendra why do we need `functions/.runtimeconfig.json`?

**Comment by bimlendra** [comment]
@hyzhak : Was just trying to avoid accidental push in future. However, I have removed that now. Can we merge this now?

**Pull request merged**